google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -1137,4 +1158,29 @@ public void downloadFileUsingRequesterPays(String projectId, String bucketName,
     blob.downloadTo(destFilePath, Blob.BlobSourceOption.userProject(projectId));
     // [END storage_download_file_requester_pays]
   }
+
+
+  /**
+   * Example of retrieving Requester pays status on a bucket.
+   */
+  public Bucket setDefaultKMSKey(String bucketName, String kmsKeyName) throws StorageException {
+    // [START storage_set_bucket_default_kms_key]
+    // Instantiate a Google Cloud Storage client
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+
+    // The name of the KMS-key to use as a default
+    // Key names are provided in the following format:
+    // 'projects/<PROJECT>/locations/<LOCATION>/keyRings/<RING_NAME>/cryptoKeys/<KEY_NAME>'
+    // String kmsKeyname = ""
```

Comment:
If you're going to add `String kmsKeyName` ("n" needs to be caps), could you also add bucketName?